statsmodels/tsa2/sarimax/model.py

```diff
@@ -0,0 +1,219 @@
+"""
+ARIMA model class.
+
+Author: Chad Fulton
+License: BSD-3
+"""
+
+import warnings
+
+from statsmodels.tsa.statespace import sarimax
+from statsmodels.tsa.statespace.tools import diff
+
+from statsmodels.tsa2.sarimax.estimators.yule_walker import yule_walker
+from statsmodels.tsa2.sarimax.estimators.burg import burg
+from statsmodels.tsa2.sarimax.estimators.hannan_rissanen import hannan_rissanen
+from statsmodels.tsa2.sarimax.estimators.innovations import (
+    innovations, innovations_mle)
+from statsmodels.tsa2.sarimax.estimators.gls import gls as estimate_gls
+from statsmodels.tsa2.sarimax.estimators.statespace import statespace
+
+from statsmodels.tsa2.sarimax.specification import SARIMAXSpecification
+from statsmodels.tsa2.sarimax.params import SARIMAXParams
+
+
+class ARIMA(sarimax.SARIMAX):
```

Comment:
I see the appeal of implementing AR/ARMA/ARIMA in terms of statespace, but these should replace the existing models completely.

Reply:
Not sure I understand what you're getting at here. It sounds like you see some conflict between these goals?

Reply:
I don't want to see arima_model.py, ar_model.py etc become orphaned files and sit there forever slowly decaying (like arma_mle, descriptivestats, mlemodel, varma_process).  So I like the new implementations, just want to remove the old ones at the same time.

Reply:
Yes, that is also my goal, once the new implementations replace all of the features of the old models. I think that is true now, with the exception of model selection in the AR model. I anticipate that model selection will be replaced by #5151 (since #4621 has stalled).